lib/runtime/procs/base.js

```diff
@@ -32,6 +33,16 @@ var base = Base.extend({
         this.logger = JuttleLogger.getLogger(this.logger_name);
     },
 
+    validate_options: function(allowedOptions) {
+        var unknown = _.difference(_.keys(this.options), allowedOptions);
+        if (unknown.length > 0) {
+            throw new this.compile_error('RT-UNKNOWN-OPTION-ERROR', {
```

Comment:
I still think we should start using `new errors.compileError()` everywhere. But maybe that's just me.